pkgs/by-name/mn/mnamer/package.nix

```diff
@@ -1,31 +1,52 @@
-{ python3Packages, fetchFromGitHub, lib }:
+{
+  python3Packages,
+  fetchFromGitHub,
+  fetchpatch,
+  lib,
+}:
 
 python3Packages.buildPythonApplication rec {
   pname = "mnamer";
-  version = "2.5.4";
+  version = "2.5.5";
+  format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "jkwill87";
     repo = "mnamer";
     rev = version;
-    sha256 = "sha256-fONQq/RboWHFuEFU7HP1ThUpSjOIlkg54c2WlMUKwuk=";
+    sha256 = "sha256-qQu5V1GOsbrR00HOrot6TTAkc3KRasBPDEU7ZojUBio=";
   };
 
+  build-system = with python3Packages; [
+    setuptools
+    setuptools-scm
+  ];
+
+  nativeBuildInputs = with python3Packages; [ pythonRelaxDepsHook ];
```

Comment:
```suggestion
```
This is not longer needed, as it is added automatically when `pythonRelaxDeps` is used

Reply:
Thanks, removed.